src/components/pets/PetsList.js

```diff
@@ -1,8 +1,11 @@
+import { useParams } from "react-router-dom";
 import PetsListNav from "./PetsListNav";
 import Pet from "./Pet";
 import "./PetsList.css";
 
 export const PetsList = ({ pets }) => {
+  const { animalType } = useParams(); //animalType will display all animals
```

Comment:
So the deal with the one failing test is that if you go to just `/pets/`, you should redirect to `/pets/cats`, rather than showing all animals